src/test/java/uk/gov/hmcts/reform/blobrouter/services/SasTokenGeneratorServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.blobrouter.services;
+
+import com.azure.storage.common.StorageSharedKeyCredential;
+import com.azure.storage.common.Utility;
+import com.azure.storage.common.implementation.StorageImplUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.exceptions.ServiceConfigNotFoundException;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@ExtendWith(MockitoExtension.class)
+class SasTokenGeneratorServiceTest {
+    private ServiceConfiguration serviceConfiguration;
+    private SasTokenGeneratorService tokenGeneratorService;
+
+    @BeforeEach
+    void setUp() {
+        StorageSharedKeyCredential storageCredentials = new StorageSharedKeyCredential(
+            "testAccountName", "dGVzdGtleQ=="
+        );
+
+        createAccessTokenConfig();
+
+        tokenGeneratorService = new SasTokenGeneratorService(
+            storageCredentials,
+            serviceConfiguration
+        );
+    }
```

Comment:
Is it possible/beneficial to move to `@BeforeAll` section - load everything statically? `storageCredentials` is automatically top level private static field. the rest can be set up once per test - also statically

Reply:
Done